lib/geokit/geocoders/bing.rb

```diff
@@ -77,7 +77,7 @@ def self.set_address_components(loc, xml)
         'Postcode1'      => 'zip',
         'Postcode2'      => 'zip',
         'RoadBlock'      => 'street',
-        'Address'        => 'address'
+        'Address'        => 'address',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.